contracts/PartnerProxy/Renewer/PartnerRenewerProxy.sol

```diff
@@ -0,0 +1,95 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+pragma solidity ^0.8.17;
+
+import {Ownable} from "@openzeppelin/contracts/access/Ownable.sol";
+import "../../Registrar/IBaseRegistrar.sol";
+import "@rsksmart/erc677/contracts/IERC677.sol";
+import "../../BytesUtils.sol";
+import "hardhat/console.sol";
+import "./IPartnerRenewerProxy.sol";
+import "../../Renewer/IBaseRenewer.sol";
+
+contract PartnerRenewerProxy is IPartnerRenewerProxy, Ownable {
+    IBaseRegistrar private _partnerRegistrar;
+    IBaseRenewer private _partnerRenewer;
+    IERC677 private _rif;
+    // sha3('renew(string,uint)')
```

Comment:
Since this is now redundant, consider removing?

Reply:
Why is it redundant?

Reply:
I guess I'm misunderstanding the comment on line 16, is it to show how the renew signature on line 17 was gotten? if so then apologies, I thought it was a piece of code you commented out and no longer needed hence why I said to remove it